frontend/src/app/firewood/forests/firewood-rules/firewood-permit-rules.component.ts

```diff
@@ -17,6 +17,7 @@ export class FirewoodPermitRulesComponent implements OnInit {
   @Input() numberOfCords: any;
   @Input() emailAddress: any;
   applicationRulesForm: FormGroup;
+  buttonDisabled: boolean;
```

Comment:
Can we just do `buttonDisabled = false;` here, since we're later setting that as the default value on line 30?

Reply:
I tried that, but when you set it to true you get the error `Type 'true' is no assignable to type 'false'` when I try to reassign it later on. Is there another way around it? I'll google around and see if I can find something.